.travis.yml

```diff
@@ -37,10 +37,22 @@ script:
   - ./gradlew copyUnitTestBuildArtifacts
 #  - ./travis-acceptance-tests.sh
 
+  - ./gradlew check -PdisablePreDex --continue --stacktrace
+  - android-wait-for-emulator
+  - adb devices
+  - adb shell input keyevent 82 &
+  - ./gradlew connectedAndroidTest -PdisablePreDex --continue --stacktrace
```

Comment:
Or more specifically, `./gradle lintProdDebug` to use only the same build as the rest of the tests.